src/main/java/at/hannibal2/skyhanni/features/misc/HideFarEntities.kt

```diff
@@ -0,0 +1,40 @@
+package at.hannibal2.skyhanni.features.misc
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.CheckRenderEntityEvent
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.utils.CollectionUtils.sorted
+import at.hannibal2.skyhanni.utils.EntityUtils
+import at.hannibal2.skyhanni.utils.LocationUtils.distanceToPlayer
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class HideFarEntities {
+    private val config get() = SkyHanniMod.feature.dev.hideFarEntities
+
+    private var ignored = emptyList<Int>()
```

Comment:
The `.map{}` function returns a list. I would have to make the list to a set. But we already have the list, why not use it directly?
![grafik](https://github.com/hannibal002/SkyHanni/assets/24389977/bf4f6d51-f65a-452c-b1d4-00a89e7f4e57)

Reply:
for the render checks. Since a contains for a set is O(1) where a contains for a list is O(n). Sure the toSet operation is not nothing but it runs only every tick where as I understand the renderchecking runs every frame for each entity,

Reply:
Did not know sets are that much faster compared to lists. In that case, for the current scenario (and looking at the refresh time vs the query time) its useful to create additional sets each refresh)

(Who had thought not saving an index saves "contians" lookups this much)